pangeo-notebook/environment.yml

```diff
@@ -65,6 +65,7 @@ dependencies:
  - jupyter-resource-usage
  - jupyter-sshd-proxy
  - kerchunk
+ - libgdal-arrow-parquet
```

Comment:
@wildintellect @jsignell, does VEDA also need other drivers (e.g. for netcdf) as well? The full list is somewhere in https://github.com/conda-forge/gdal-feedstock/blob/a28b550c8887cad64fd5fc43ad5a64d6c9852398/recipe/meta.yaml#L554-L567 (look around other parts of that file too).

Reply:
That's the most important one for better vector support. We may come back later after some more experimentation with grib2 that @hrodmn has been doing but that's not common yet the way geoparquet is.